ui/e2e/features/basic_interactions.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Basic Interactions
+
+Background:
+    Given I am on the homepage
+
+# Scenario: I can start producing
+#     When I start the producer
```

Comment:
How come so many of these are commented out?

Reply:
Andy is working on these now - they will follow (these are tests being run in browser). We have the equivalent testing out of browser, so I am not too worried

Reply:
Ok, happy for these to be uncommented in a future PR